### PR TITLE
Add AWS and modify image to use cloud-init to create projects

### LINKED ISSUE #4: [TO-DO] Move Strapi project generation to cloud-init
Should move initial project creation to the cloud-init so we don't need to update images to use latest versions.

Need to discuss with @alexandrebodin and @pierreburgy about how best to handle this as we also want to avoid users creating projects that already have quick hot-fixes.

Should we stick to the existing system where each image is specifically constructed to use one version  or set it to use the latest?

### PR DESCRIPTION
**Pull request by derrickmehaffy**
Short TLDR of what has changed:

- Added AWS image scripts for the AWS Marketplace
- PG Database and user are now set in the cloud-init per-instance script
- Strapi project is created, built, ran, and set to start on boot in the cloud-init per-instance script
- Moved Strapi project location to `/srv/strapi/strapi-development` as to not confuse people if they want to run dev and prod on the same server
- Added additional message to the MOTD to tell the user if Strapi is still currently being installed
- Simplified initial image creation with the drawback of first VM start taking a bit longer (but more secure as users no longer need to change PG password themselves)
- Custom UUID generation no longer required

After this is merged I will have a linked PR into the Strapi docs to modify them relative to these updates, likewise once Strapi is published on the AWS Marketplace I will create related docs for AWS as well (certain parts are a bit different than DO)

fix: #4

### CODE REVIEW

**Comment by derrickmehaffy** [comment]
Am performing final testing so please wait to merge until I finish all testing.

**Comment by derrickmehaffy** [comment]
Tests have passed, once merged I will talk with @pierreburgy about updating the DO image and start working on the AWS Marketplace (already discussed that he needs to make that account with AWS)

Will start work on the DO doc changes now.

**Comment by derrickmehaffy** [comment]
Image has been regenerated and sent to DigitalOcean

File: aws-ami/configs/strapi/index.html
Comment (alexandrebodin) [inline aws-ami/configs/strapi/index.html]: We are going to make a breaking change in beta.20 to allow a user to delete the files in the public folder. You can check the PR here to know more https://github.com/strapi/strapi/pull/5444

**Comment by alexandrebodin** [comment]
@derrickmehaffy  does the link in the index page work ?

**Comment by derrickmehaffy** [comment]
I need to test, got side tracked. I created that work around for beta.18.x and I think a fix was made to pull from the internal variable

**Review by alexandrebodin — APPROVED** [review]
LGTM, Will need to test once published :)

**Pull request merged**
